packages/frames.js/src/getFrameFlattened.ts

```diff
@@ -24,11 +24,11 @@ export function getFrameFlattened(frame: Frame): FrameFlattened {
       }),
       {}
     ),
-    ...(frame.accepts?.map(
-      ({ id, version }) =>
-        `<meta name="of:accepts:${id}" content="${version}"/>`
-    ) ?? []),
   };
 
+  for (const accept of frame.accepts ?? []) {
+    metadata[`of:accepts:${accept.id}`] = accept.version;
+  }
+
```

Comment:
Should either add all dynamic keys in here through mutation or avoid it completely